app/build.gradle

```diff
@@ -1,12 +1,13 @@
 apply plugin: 'com.android.application'
+apply plugin: 'org.jetbrains.kotlin.android'
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 33
     buildToolsVersion "30.0.3"
     defaultConfig {
         applicationId "com.glia.exampleapp"
         minSdkVersion 24
-        targetSdkVersion 30
+        targetSdkVersion 33
```

Comment:
If I'm not mistaken, `targetSdkVersion` increase may cause issues for the integrators and we decided to do this as later as possible, don't we?